build.py

```diff
@@ -0,0 +1,357 @@
+                          "# official releases currently.",
+                          "-DPXR_ENABLE_GL_SUPPORT=ON",
+                          "# OIIO is used for loading image textures in Hydra Storm / Embree renderers.",
+                          "-DPXR_BUILD_OPENIMAGEIO_PLUGIN=ON",
+                          "# USD 22.03 does not support OCIO 2.x",
+                          "# Tracking ticket https://github.com/PixarAnimationStudios/USD/issues/1386",
+                          "-DPXR_BUILD_OPENCOLORIO_PLUGIN=OFF",
+                          "-DPXR_ENABLE_PTEX_SUPPORT=OFF",
+                          "-DPXR_BUILD_USD_TOOLS=OFF",
+                          "-DCMAKE_DEBUG_POSTFIX=_d",
+                          "-DBUILD_SHARED_LIBS=ON",
+                          f"-DTBB_INCLUDE_DIRS={blender_libs_dir}/tbb/include",
+                          f"-DTBB_LIBRARIES={blender_libs_dir}/tbb/lib/{LIBPREFIX}tbb{SHAREDLIBEXT}",
+                          f"-DTbb_TBB_LIBRARY={blender_libs_dir}/tbb/lib/{LIBPREFIX}tbb{SHAREDLIBEXT}",
+                          f"-DTBB_tbb_LIBRARY_RELEASE={blender_libs_dir}/tbb/lib/{LIBPREFIX}tbb{SHAREDLIBEXT}",
+                          "# USD wants the tbb debug lib set even when you are doing a release build",
+                          "# Otherwise it will error out during the cmake configure phase.",
+                          f"-DTBB_LIBRARIES_DEBUG={blender_libs_dir}/tbb/lib/{LIBPREFIX}tbb{SHAREDLIBEXT}",
+                          #################################################
+                          f"-DBoost_INCLUDE_DIR={blender_libs_dir}/boost/include",
+                          f"-DMaterialX_DIR={bin_dir}/USD/install/lib/cmake/MaterialX",
+                          ]
+
+
+        try:
+            _cmake(usd_dir, compiler, jobs, build_var, [
+                *USD_EXTRA_ARGS,
+                f'-DCMAKE_INSTALL_PREFIX={bin_dir / "USD/install"}',
+            ])
+
+        finally:
+            print("Reverting USD repo")
+            check_call('git', 'checkout', '--', '*')
+            check_call('git', 'clean', '-f')
+
+    finally:
+        os.chdir(cur_dir)
+
+
+def hdrpr(blender_libs_dir, bin_dir, compiler, jobs, clean, build_var):
+    print_start("Building HdRPR")
+
+    hdrpr_dir = repo_dir / "RadeonProRenderUSD"
+    usd_dir = bin_dir / "USD/install"
+
+    if clean:
+        rm_dir(hdrpr_dir / "build")
+
+    os.environ['PXR_PLUGINPATH_NAME'] = str(usd_dir / "lib/usd")
+
+    PYTHON_SHORT_VERSION_NO_DOTS = 310
+    BOOST_VERSION_SHORT = 180
+    BOOST_COMPILER_STRING = "-vc142"
+
+    PYTHON_POSTFIX = "_d" if build_var == 'debug' else ""
+    OPENEXR_VERSION_POSTFIX = "_d" if build_var == 'debug' else ""
+    LIBEXT = ".lib" if OS == 'Windows' else ".a"
+    LIBPREFIX = "" if OS == 'Windows' else "lib"
+    SHAREDLIBEXT = ".lib" if OS == 'Windows' else ""
+    PYTHON_EXTENSION = ".exe" if OS == 'Windows' else ""
+
+
+    os.add_dll_directory("C:/GPUOpen/Blender/lib/win64_vc15/imath/bin")
```

Comment:
Seems blender_libs_dir should be used here instead C:/GPUOpen/Blender/lib/win64_vc15